### ISSUE TITLE
How to avoid shared build path between project folders

### ISSUE THREAD

**Issue by antonkulaga** [mpeltonen/sbt-idea#223]
When I create a multiproject in sbt ( https://gist.github.com/antonkulaga/5506349 ) and than do gen-idea on it (I use version 1.4.0) I get 

"scala: Output path /home/antonkulaga/denigma/gene-app/modules/gene-actors/project/target/idea-test-classes is shared between: Module 'gene-actors-build' tests, Module 'semantic-data-build' tests
Output path /home/antonkulaga/denigma/gene-app/modules/gene-actors/project/target/idea-classes is shared between: Module 'gene-actors-build' production, Module 'semantic-data-build' production Currently external Scala compiler prohibits output path sharing."

I think it is more likely my fault than a bug in gen-idea, but could please tell me how to avoid it?

**Comment by mpeltonen** [comment]
Your project structure is a bit hard to follow because of traits, but the reason might be that you don't have aggregate project and you've had to run 'gen-idea' separately on both projects?

I'd suggest to make the project hierarchy definition a bit more clear like in this sample: http://www.scala-sbt.org/release/docs/Getting-Started/Multi-Project#defining-projects-in-a-scala-file and either make 'geneActors' aggregate 'data', or create third 'root' project like in the example and put both of the child projects into separate subdirectories.

But this is just guessing at this point. Do you have the whole project somewhere in Github to actually try gen-idea on it?

**Comment by rparree** [comment]
I have the same problem (plus another, i will post separate). I have a public project on github that illustrates the problem: https://github.com/rparree/camel-demos

Am using SBT 0.13.0
IntelliJ Cardea IU-132.435

I need to manually remove the *-build modules in order to leave the project settings in case i made any changes to it.

**Comment by rparree** [comment]
It had to do with the `project` directory inside a submodule. (`simple-jms` in my particular example)
